spec/features/view_bookings_spec.rb

```diff
@@ -0,0 +1,46 @@
+require "rails_helper"
+
+RSpec.feature "Viewing past Bookings" do
+  before :each do
+    FactoryGirl.create(:booking_with_passengers)
+    FactoryGirl.create(:flight)
+    FactoryGirl.create(:plane)
+    FactoryGirl.create(:airline)
+    OmniAuth.config.test_mode = true
+    OmniAuth.config.add_mock(:facebook, {:uid => "123456"})
+    visit root_path
+    click_link "Login"
+    click_link "Sign in with Facebook"
+  end
+  scenario "Logged in user wants to view past bookings" do
+    visit mybooking_path
+    expect(page).to  have_content ("Passenger")
+    expect(page).to  have_content ("Cost")
+    expect(page).to  have_content ("Booking Id")
+    expect(page).to  have_content ("Ticket No")
+    expect(page).to  have_content ("Time")
+  end
+  scenario "Logged in user wants to edit past booking" do
+    visit findme_path
+    expect(page).to  have_content ("Booking Id")
```

Comment:
`(...)` interpreted as grouped expression.<br>Put one space between the method name and the first argument.